docs/manual/installation/install-contao.de.md

```diff
@@ -106,11 +106,40 @@ Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
 {{% /notice %}}
 
 
-### Datenbanktabellen aktualisieren
+### Datenbank erstellen
+
+Nach der Installation kannst du die Datenbank Aktualisierung über das [Contao-Installtool](/de/installation/contao-installtool/) 
+durchführen. Ab Contao 4.9 steht dir hierzu auch der folgende Befehl auf der Kommandozeile zur Verfügung:
```

Comment:
This is not quite correct. This sounds like you can create a database in the Contao Install Tool, which is not possible. This needs to go into the next section.